src/main/java/org/opentripplanner/transit/service/StopModel.java

```diff
@@ -33,247 +29,254 @@ public class StopModel implements Serializable {
 
   private static final Logger LOG = LoggerFactory.getLogger(StopModel.class);
 
-  /** Parent stops **/
-  private final Map<FeedScopedId, Station> stationById = new HashMap<>();
-  /**
-   * Optional level above parent stops (only supported in NeTEx)
-   */
-  private final Map<FeedScopedId, MultiModalStation> multiModalStationById = new HashMap<>();
-  /**
-   * Optional grouping that can contain both stations and multimodal stations (only supported in
-   * NeTEx)
-   */
-  private final Map<FeedScopedId, GroupOfStations> groupOfStationsById = new HashMap<>();
-
-  private final Map<FeedScopedId, TransitStopVertex> transitStopVertices = new HashMap<>();
+  private final Map<FeedScopedId, Stop> stopsById;
+  private final Map<FeedScopedId, Station> stationById;
+  private final Map<FeedScopedId, MultiModalStation> multiModalStationById;
+  private final Map<FeedScopedId, GroupOfStations> groupOfStationsById;
+  private final Map<FeedScopedId, FlexStopLocation> flexStopsById;
+  private final Map<FeedScopedId, FlexLocationGroup> flexStopGroupsById;
 
   /** The density center of the graph for determining the initial geographic extent in the client. */
-  private Coordinate center = null;
+  private final WgsCoordinate stopLocationCenter;
 
-  private final Map<FeedScopedId, FlexStopLocation> locationsById = new HashMap<>();
+  private transient StopModelIndex index;
 
-  private final Map<FeedScopedId, FlexLocationGroup> locationGroupsById = new HashMap<>();
+  public StopModel() {
+    System.err.println("StopModel.StopModel(): " + Objects.hashCode(this));
+    this.stopsById = Map.of();
+    this.stationById = Map.of();
+    this.multiModalStationById = Map.of();
+    this.groupOfStationsById = Map.of();
+    this.flexStopsById = Map.of();
+    this.flexStopGroupsById = Map.of();
+    this.stopLocationCenter = null;
+  }
 
-  private transient StopModelIndex index;
+  public StopModel(StopModelBuilder builder) {
+    System.err.println(
```

Comment:
Use regular logging

Reply:
This is still in Draft, I some times add System.out/err for debugging  - witch I hopefully remove from history before review.